### ISSUE TITLE
Techs H91 事例2eの文章が原文と異なる内容になっている

### ISSUE THREAD

**Issue by bakera** [waic/wcag20#262]
(A11y Slackでmackeyさんよりご指摘いただいた内容を転載)
https://waic.jp/docs/WCAG-TECHS/H91

>事例 2e
>HTML 4.01 及び XHTML 1.x において、見出し要素はセクションの先頭をマークするだけであることに注意すること。見出し要素をセクションコンテンツに明示的に関連付けるためのマークアップが存在しないため、次の見出し要素が出現するまで、見出しは後に続くすべてのコンテンツに適用されると見なされる。
>コード例:
><input type="image" src="save.gif" alt="保存" title="保存" /> 

https://www.w3.org/TR/WCAG20-TECHS/H91.html
>Example 2e
>Example 2e clarifies how the user agent determines the name if the author specifies both the 'alt' and 'title' attributes of the input element. In this case, the user agent uses the 'alt' attribute ("Save") and ignores the 'title' attribute.
>Example Code:
><input type="image" src="save.gif" alt="save" title="save the file" />```

↑英語を単純に翻訳すると
「例2eは、入力要素の 'alt'属性と 'title'属性の両方を指定した場合、ユーザーエージェントが名前をどのように決定するかを明確にしています。この場合、ユーザーエージェントは 'alt'属性（ "保存"）を使用し、 'title'属性を無視します。」
になるはずが突然見出し要素の話になっています

**Comment by bakera** [comment]
現状のものは、H69の事例3の訳文が入ってしまっているようです。
https://waic.jp/docs/WCAG-TECHS/H69.html

あらためて翻訳して日本語を入れる必要がありそう。
